test/quantization/test_quantize_fx.py

```diff
@@ -104,7 +104,7 @@ class BinaryOpNonQuantizedInput(torch.nn.Module):
     def __init__(self, binary_op, ibinary_op, is_inplace, is_scalar):
         super().__init__()
         self.is_scalar = is_scalar
-        self.op = ibinary_op if is_inplace else binary_op
+        self.op = ibinary_op if ibinary_op and is_inplace else binary_op
```

Comment:
optional: what's the difference between binary_op or ibinary_op?  Would be great if there was a comment

Reply:
oh that means inplace binary op